pkg/storage/unified/resource/go.mod

```diff
@@ -2,20 +2,249 @@ module github.com/grafana/grafana/pkg/storage/unified/resource
 
 go 1.23.1
 
+replace (
+	github.com/grafana/grafana => ../../../..
```

Comment:
these seem like they shouldn't be needed 🤔

Reply:
Hmm... that seems true. I don't remember why I added these in particular, but I do remember that without these replaces, I wasn't able to make the change.

Reply:
Ah yes. Without this, `make update-workspace` causes the module to be looked up and downloaded:

```
go: found github.com/grafana/grafana/pkg/infra/tracing in github.com/grafana/grafana v6.1.6+incompatible
```

This obviously causes problems down the line.

It does seem to me like workspaces _should_ support this but it seems they don't. I think part of the problem goes back to the versioning issue (our Go modules aren't versioned properly).